components/accordion/Accordion.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
+import styles from "./Accordion.module.css";
+
+export type Content = {
```

Comment:
You propably should rename this Prop type for better understanding later on :)

```suggestion
export type AccordionProps = {
```